api/accounts/getAccount.ts

```diff
@@ -1,6 +1,6 @@
-import * as auth from './authenticate';
```

Comment:
I don't like this naming scheme because it has the word `account` multiple times, redundantly.